nengo_viz/viz.py

```diff
@@ -180,8 +182,16 @@ def get_uid(self, obj):
     def start(self, port=8080, browser=True):
         """Start the web server"""
         nengo_viz.server.Server.viz = self
-        nengo_viz.server.Server.start(port=port, browser=browser)
+        # asynch must be true to handle websockets
+        return nengo_viz.server.Server.start(
+            port=port, browser=browser, asynch=True)
 
     def create_sim(self):
         """Create a new Simulator with this configuration"""
-        return VizSim(self)
+        vizsim = VizSim(self)
+        self.vizsims.append(vizsim)
+        return vizsim
+
+    def cleanup(self):
+        for vizsim in self.vizsims:
+            vizsim.finish()
```

Comment:
We don't have anything like this in `master`, but it seems unnecessary because `VizSim.finish()` is called by `SimControl.finish()` which in turn is called at the end of `server.ws_viz_component`. The `VizSim` threads seem to exit cleanly on my machine in any case, and if they don't exit cleanly on other machines they will be reported as zombie threads.